pkg/ingester/instance.go

```diff
@@ -63,6 +63,7 @@ var (
 
 type instance struct {
 	cfg        *Config
+	limits     *validation.Overrides
```

Comment:
This should already be covered here:
https://github.com/grafana/loki/blob/main/pkg/ingester/instance.go#L83